diff --git a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
index 508494a..0e9fa41 100644
--- a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
+++ b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
@@ -155,6 +155,10 @@ static NSString *RCTGetStorageDirectory()
     static NSString *storageDirectory = nil;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
+#if TARGET_OS_TV
+      RCTLogWarn(
+        @"Persistent storage is not supported on tvOS, your data may be removed at any point.");
+#endif
       storageDirectory = RCTCreateStorageDirectoryPath(RCTStorageDirectory);
     });
     return storageDirectory;
@@ -499,11 +503,6 @@ RCT_EXPORT_MODULE()
 {
     RCTAssertThread(RCTGetMethodQueue(), @"Must be executed on storage thread");
 
-#if TARGET_OS_TV
-    RCTLogWarn(
-        @"Persistent storage is not supported on tvOS, your data may be removed at any point.");
-#endif
-
     NSError *error = nil;
     if (!RCTHasCreatedStorageDirectory) {
         _createStorageDirectory(RCTGetStorageDirectory(), &error);
