diff --git a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
index 508494a..77f8147 100644
--- a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
+++ b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
@@ -500,7 +500,7 @@ RCT_EXPORT_MODULE()
     RCTAssertThread(RCTGetMethodQueue(), @"Must be executed on storage thread");
 
 #if TARGET_OS_TV
-    RCTLogWarn(
+    RCTLogInfo(
         @"Persistent storage is not supported on tvOS, your data may be removed at any point.");
 #endif
 
